config/routes.rb

```diff
@@ -7,40 +7,40 @@
     # How (or even if) API versioning will be implemented is still an open question. We can defer
     # the choice until we need to expose the API to clients which we don't control.
     scope module: :v1, constraints: ApiConstraints.new(version: 1, default: true) do
-      resources :sections, only: [:index, :show], constraints: { id: /\d{1,4}/ } do
-        scope module: 'sections', constraints: { section_id: /\d{1,2}/, id: /\d+/ } do
+      resources :sections, only: [:index, :show], constraints: { id: /\d+/ } do
+        scope module: 'sections', constraints: { section_id: /\d+/, id: /\d+/ } do
           resource :section_note, only: [:show, :create, :update, :destroy]
           resources :search_references, only: [:show, :index, :destroy, :create, :update]
         end
       end
 
-      resources :chapters, only: [:index, :show], constraints: { id: /\d{2}/ } do
+      resources :chapters, only: [:index, :show], constraints: { id: /\d+/ } do
         member {
           get :changes
         }
 
-        scope module: 'chapters', constraints: { chapter_id: /\d{2}/, id: /\d+/ } do
+        scope module: 'chapters', constraints: { chapter_id: /\d+/, id: /\d+/ } do
           resource :chapter_note, only: [:show, :create, :update, :destroy]
           resources :search_references, only: [:show, :index, :destroy, :create, :update]
         end
       end
 
-      resources :headings, only: [:show], constraints: { id: /\d{4}/ } do
+      resources :headings, only: [:show], constraints: { id: /\d+/ } do
         member {
           get :changes
         }
 
-        scope module: 'headings', constraints: { heading_id: /\d{4}/, id: /\d+/ } do
+        scope module: 'headings', constraints: { heading_id: /\d+/, id: /\d+/ } do
           resources :search_references, only: [:show, :index, :destroy, :create, :update]
         end
       end
 
-      resources :commodities, only: [:show], constraints: { id: /\d{10}/ } do
+      resources :commodities, only: [:show], constraints: { id: /\d+/ } do
         member {
           get :changes
         }
 
-        scope module: 'commodities', constraints: { commodity_id: /\d{10}/, id: /\d+/ } do
```

Comment:
wait what? yeah I would change that, as these restrictions should be there I think